WeaveDemoText.txt

```diff
@@ -0,0 +1 @@
+This is some text for the weave notebook demo.
```

Comment:
Can we create this file in the demo notebook then delete it at the end of the notebook instead?

Reply:
Yes, that is an excellent idea.